manifests/charts/istiod-remote/values.yaml

```diff
@@ -47,6 +47,8 @@ defaults:
   volumeMounts: []
   # Additional volumes to the istiod pod
   volumes: []
+  # Inject initContainers into the istiod pod
+  initContainers: []
```

Comment:
why change this?

Reply:
That was made by ‘make gen’…

Reply:
let me double check this, it's weird.

Reply:
`istiod-remote` is a makefile copypaste of `istio-discovery` for the most part, this makes sense.

Reply:
`istiod-remote` is a makefile copypaste of `istio-discovery` for the most part, this makes sense.

Reply:
https://github.com/istio/istio/blob/c4962f77275ac7213a26a280a08af941a78d6bbd/Makefile.core.mk#L380